### ISSUE TITLE
[FEATURE REQUEST] Add Pylint configuration to epicli devcontainer

### ISSUE THREAD

**Issue by seriva** [epiphany-platform/epiphany#2531]
**Is your feature request related to a problem? Please describe.**
Pylint is installed in the devcontainer but lacks configuration.

**Describe the solution you'd like**
Add default Pylint configuration so it can be used.

**Describe alternatives you've considered**
None

**Additional context**
The decision for Pylint as linter was already made some time back but never fully applied for use in the project.

---

**DoD checklist**

* [x] Changelog updated
* [x] COMPONENTS.md updated / doesn't need to be updated
* [x] Schema updated / doesn't need to be updated
* [ ] Feature has automated tests
* [ ] Automated tests passed (QA pipelines)
  * [ ] apply
  * [ ] upgrade
* [ ] Idempotency tested
* [x] Documentation added / updated / doesn't need to be updated
* [x] All conversations in PR resolved
* [x] Solution meets requirements and is done according to design doc
* [x] Usage compliant with license
* [x] Backport tasks created / doesn't need to be backported

**Comment by przemyslavic** [comment]
:heavy_check_mark: I have rebuilt the devcontainer and ran a pylint check. To avoid the `bash: /workspaces/epiphany/core/src/epicli/test_results/pylint_results.xml: No such file or directory` error, either create the `test_results` folder in the `/workspaces/epiphany/core/src/epicli` path or run unit tests first (discussed with @seriva not to fix it right now).
I've also deployed the clusters: `AWS` / `Azure` x `Ubuntu` / `RHEL` / `CentOS` with all components enabled (at least 1 vm) as there were some changes in playbooks and did not run into any problems.
